commands/root.go

```diff
@@ -50,8 +50,8 @@ func init() {
 
 	RootCmd.PersistentFlags().String("dbpath", "", "/path/to/sqlite3 or SQL connection string")
 	_ = viper.BindPFlag("dbpath", RootCmd.PersistentFlags().Lookup("dbpath"))
-	pwd := os.Getenv("PWD")
-	viper.SetDefault("dbpath", filepath.Join(pwd, "go-exploitdb.sqlite3"))
+	dbpath := os.Getenv("DB_PATH")
```

Comment:
Do we have an default value for this if not specified ?

Reply:
If the env is missing then the current directory will automatically be used for the db file. Have kept it consistent with what the library already had. Just replaced the env "PWD" with "DB_PATH".

It is followed by 
`viper.SetDefault("dbpath", filepath.Join(dbpath, "go-exploitdb.sqlite3"))`